themes/custom/droopler_theme/templates/layout/page.html.twig

```diff
@@ -138,16 +138,22 @@
       <div class="{{ container }}">
         {{ page.footer_main }}
         {% if page.footer_first or page.footer_second or page.footer_third or page.footer_fourth %}
-          <div class="site-footer__top clearfix">
-            {{ page.footer_first }}
-            {{ page.footer_second }}
-            {{ page.footer_third }}
-            {{ page.footer_fourth }}
-          </div>
-        {% endif %}
-        {% if page.footer_fifth %}
-          <div class="site-footer__bottom">
-            {{ page.footer_fifth }}
+          <div class="site-footer-blocks row">
+            {% for i, region in filled_regions %}
+              {% if filled_regions|length == i + 1  %}
+                {% if (i + 1) % 2 == 1 %}
```

Comment:
You can use twig tests here:
{% if i is odd %}

also you can simplify code
{% set classes = i is odd ? 'col' : ['col-6', 'col-xl'] %}